src/Sulu/Bundle/ContentBundle/.gitignore

```diff
@@ -3,6 +3,7 @@
 /composer.phar
 /composer.lock
 /.sass-cache/
+/Resources/app/cache
```

Comment:
I don't think so... Tests are still running, so certain files still need to be ignored. However, there are other files that could be removed from all the bundles, because they are orphaned anyway (like the  changelog or composer.json).